src/plugins/discover/public/components/discover_grid/constants.ts

```diff
@@ -11,10 +11,10 @@ import { EuiDataGridStyle } from '@elastic/eui';
 // data types
 export const kibanaJSON = 'kibana-json';
 export const GRID_STYLE = {
-  border: 'all',
+  border: 'horizontal',
   fontSize: 's',
-  cellPadding: 's',
-  rowHover: 'none',
+  cellPadding: 'm',
```

Comment:
If the intent here is to begin adopting some aspects of the newly proposed [Discover design refresh](https://www.figma.com/file/zxhZnjEOmgodwQwQlkZ6pZ/Data-grid-improvements-v2?type=design&node-id=181%3A587864&mode=design&t=a0Rm7Gji4IL10CaH-1), this padding will need to be increased to match the designs.

 ```suggestion
  cellPadding: 'l',
```